README.md

```diff
@@ -69,104 +85,226 @@ git remote add upstream https://github.com/geirev/EnKF_sampling
 popd
 ```
 
-If you are new to git, read the section <a href="#git-instructions">Git instructions</a>
+If you are new to Git, read the section <a href="#git-instructions">Git instructions</a>
 
+## 2. Required Packages
+
+### Linux
 
-## 2. Install blas, lapack, libfftw3, and Fortran95
-<a href="#Mac-OS-X-installation">Additional instructions for installation on Mac OS X</a>
 ```bash
 sudo apt-get update
 sudo apt-get install libblas-dev liblapack-dev libfftw3-dev gfortran
 ```
 
-## 3. Compile the EnKF_sampling library
+### Mac
+
+```bash
+brew install gcc fftw openblas lapack
+```
+
+**Note:** You must have [Homebrew](https://brew.sh/) installed to install
+packages using `brew`
+
+## 3. Compile the `EnKF_sampling` library
+
+Navigate to the `lib` folder of the `EnKF_sampling` repository:
+
 ```bash
 cd EnKF_sampling/lib
+```
+
+then compile and place all the `.o` files as well as `libanalysis.a` into
+the `build` directory of the `EnKR_seir` repository using:
+
+```bash
 make BUILD=../../EnKF_seir/build
 ```
-This will put all the .o files as well as libanalysis.a in the same dir as you will use when compiling EnKF_seir
 
-## 4. Compile the EnKF_analysis library
+## 4. Compile the `EnKF_analysis` library
+
+Navigate to the `lib` folder of the `EnKF_analysis`:
+
 ```bash
 cd EnKF_analysis/lib
+```
+
+then compile and place all the `.o` files as well as `libanalysis.a` into the
+`build` directory of the `EnKR_seir` repository using:
+
+```bash
 make BUILD=../../EnKF_seir/build
 ```
-This will put all the .o files as well as libanalysis.a in the same dir as you will use when compiling EnKF_seir.
-Note that EnKF_analysis depends on EnKF sampling.
 
-## 5. cd EnKF_seir/src
-Compile and install executable in target install dir which defaults to BINDIR=/$HOME/bin
+**Note:** The `EnKF_analysis` repository depends on the `EnKF_sampling`
+repository and therefore must be compiled second!
+
+## 5. Compile the `EnKR_seir` library
+
+### Linux
+
+Navigate to the `src` folder of the `EnKF_seir` repository:
+
+```bash
+cd EnKF_seir/src
+```
+
+then compile and install the executable in the target directory, defaulting to
+`$HOME/bin`:
+
+```bash
+make BINDIR=$HOME/bin
+```
+
+### Mac
+
+Navigate to the `src` folder of the `EnKF_seir` repository:
+
+```bash
+cd EnKF_seir/src
+```
+
+then edit the following line in `EnKF_seir/src/makefile` from:
+
+```bash
+LIBS = ./libsampling.a ./libenkfanalysis.a -llapack -lblas -llapack /usr/lib/x86_64-linux-gnu/libfftw3.so.3
+```
+
+to:
+
+```bash
+LIBS = ./libsampling.a ./libenkfanalysis.a -llapack -lblas -llapack /usr/local/lib/libfftw3.a
```

Comment:
I had to use `/usr/local/lib/libfftw3.a` instead of `-lfftw3` as I installed `fftw` using `brew install fftw`

cc @clairevalva